modules/library/jdbc/src/main/java/org/geotools/jdbc/KeysFetcher.java

```diff
@@ -605,4 +622,61 @@ public Object getNext(Connection cx) throws IOException, SQLException {
             }
         }
     }
+
+    /**
+     * For PK columns that have sequence on which the sequence value should must be fetched before
+     * the INSERT operation
```

Comment:
Missing [full stop](http://www.sussex.ac.uk/informatics/punctuation/stopsandmarks/full).

Reply:
I fix the description